extension/src/test/e2e/extension.test.ts

```diff
@@ -87,7 +88,7 @@ describe('Experiments Table Webview', function () {
       async () => {
         await webview.expandAllRows()
         const currentRows = await webview.row$$
-        return currentRows.length >= initialRows + epochs
+        return currentRows.length >= initialRows + epochs + finalExperimentRow
       },
       { interval: 5000, timeout: 180000 }
     )
```

Comment:
The e2e test appears to be filing both locally and remotely: 
```
dvc:test-e2e: [0-0] Error in "Experiments Table Webview.should update with a new row for each checkpoint when an experiment is running"
dvc:test-e2e: Error: expect(received).toStrictEqual(expected) // deep equality
dvc:test-e2e:
dvc:test-e2e: Expected: 22
dvc:test-e2e: Received: 23
dvc:test-e2e:     at Context.<anonymous> (file:///Users/juliegalvan/Desktop/workspace/vscode-dvc/extension/src/test/e2e/extension.test.ts:102:30)
dvc:test-e2e:     at processTicksAndRejections (node:internal/process/task_queues:96:5)
dvc:test-e2e: [0-0] Error in ""after all" hook in "{root}""
dvc:test-e2e: Error: Timeout of 60000ms exceeded. The execution in the test " "after all" hook in "{root}"" took too long. Try to reduce the run time or increase your timeout for test specs (https://webdriver.io/docs/timeouts).
```

Reply:
I ran it locally with the wrong version of DVC... 🤸🏻. Fixed now.